pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -317,7 +317,9 @@ def test_take_fill_value_with_timezone(self):
 
 
 class TestDatetimeIndex:
-    @pytest.mark.parametrize("null", [None, np.nan, pd.NaT])
+    @pytest.mark.parametrize(
+        "null", [None, np.nan, np.datetime64("NaT"), pd.NaT, pd.NA]
```

Comment:
should create a fixture like this (similar to nulls_fixture), but this is more datetime specialized, but for another pass.